integration-tests/pages/reviewer/notCompletedIncidentsPage.js

```diff
@@ -38,24 +33,15 @@ const incidentsPage = () =>
           .then(link => link.match(/\/(.*?)\/view-statements/)[1]),
     }),
 
-    getCompleteRow: i => ({
-      date: () => completeCol(i, 0),
-      prisoner: () => completeCol(i, 1),
-      prisonNumber: () => completeCol(i, 2),
-      reporter: () => completeCol(i, 3),
-      viewStatementsButton: () => completeCol(i, 4).find('a'),
-      reportId: () =>
-        completeCol(i, 4)
-          .find('a')
-          .invoke('attr', 'href')
-          .then(link => link.match(/\/(.*?)\/your-statement/)[1]),
-    }),
+    getCountOfNotCompleteReports: () => {
+      return cy.task('getCountOfNotCompleteReports', 'COMPLETE')
```

Comment:
tying this to a specific page really reduces the usability of this... tasks sit outside of the functionality of a page so would just use it directly in specs.

Reply:
yes I noticed that approach had been used in integration-tests/integration/createIncident/form-submit.spec.js lines 70, 95